src/test/java/com/snack/news/domain/topic/PublishedCorpTopicTest.java

```diff
@@ -0,0 +1,17 @@
+package com.snack.news.domain.topic;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PublishedCorpTopicTest {
+
+	@ParameterizedTest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4677809)